vm/src/sector/post.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use crate::{ActorID, Randomness, RegisteredProof, SectorID, SectorNumber};
+use cid::Cid;
+
+// TODO check if this can be changed to an array
+pub type ChallengeTicketsCommitment = Vec<u8>;
+pub type PoStRandomness = Randomness;
+pub type PartialTicket = [u8; 32];
+
+/// PoStVerifyInfo is the structure of all the information a verifier
+/// needs to verify a PoSt.
+pub struct PoStVerifyInfo {
+    pub randomness: PoStRandomness,
+    pub candidates: Vec<PoStCandidate>,
+    pub proofs: Vec<PoStProof>,
+    pub eligible_sectors: Vec<SectorInfo>,
+    pub prover: ActorID,
+    pub challenge_count: u64,
+}
+
+// TODO docs
+pub struct SectorInfo {
+    /// Used when sealing - needs to be mapped to PoSt registered proof when used to verify a PoSt
+    pub proof: RegisteredProof,
+    pub sector_number: SectorNumber,
+    pub sealed_cid: Cid,
+}
+
+// TODO docs
```

Comment:
I notice we have similar types in `ticket.rs` can we remove them there or are they used differently?

Reply:
yeah, I had a look, either they are different types or the ones that existed are now out of date, either way we will find out and remove if there is the connection (inevitably will not be missed later)